core/server/data/migrations/versions/4.39/2022-03-07-14-37-add-members-cancel-events-table.js

```diff
@@ -0,0 +1,8 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('members_cancel_events', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id', cascadeDelete: true},
+    from_plan: {type: 'string', maxlength: 255, nullable: true},
```

Comment:
What is the rationale around `from_plan` being nullable? Is that something that can happen? 🤔

Reply:
Good point, it can't happen so I've moved `from_plan` so that it can't be nullable, thanks!